### PR TITLE
Upgrade model training to fastai v2

### LINKED ISSUE #214: Upgrade `fastai` to version 2
Similar to #213, the model training (_i.e._ `2_FastAI_StarterScript.ipynb`) depends on fastai v1, which complicates the setup of a working environment and forces us to use outdated libraries. 

The goal of this issue is to upgrade `fastai` from v1 to v2. 

Depends on #213

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
The goal of this PR is to figure out how to upgrade the model training code in `2_FastAI_StarterScript.ipynb` to use fastai v2 (without fastaudio) and attain equivalent performance. After multiple attempts, I settled on the approach of writing two notebooks in parallel:

1. **`fastai-migration-notes-v1.ipynb`:** This is mostly code copied from `2_FastAI_StarterScript.ipynb`, but with additional exploration on my part (_e.g._ to confirm certain config values). I also output the data in this notebook so I can compare and contrast with the equivalent code in the second notebook. 
2. **`fastai-migration-notes-v2.ipynb`:** This notebook demonstrates model training using fastai v2 without fastaudio. I was able to get an accuracy on the validation set (20%) up to 94.9%, which is slightly higher than the 93.7% achieved in the original `2_FastAI_StarterScript.ipynb` notebook. While I don't know how much we can rely on these performance gains, my goal was to achieve similar (or greater) performance to make sure I hadn't introduced a regression. I believe this goal was met. 

Fixes #214

### CODE REVIEW

**Comment by pastorep** [comment]
Super nice work to get us moving away from deprecated fastaudio package and toward standard torchaudio package. Especially cool to see your experiments proving viability. 

I wonder if it makes sense to stick the notebooks in a notes/ folder, to accentuate that no future dev will run these notebooks. I ack that they're already titled *-notes*.ipynb, but still could be confusing since they're in same directory as our "prod" code.

**Comment by pastorep** [comment]
Although it does seem that fastai-migration-notes-v2.ipynb may be a reasonable plug-in replacement for 2_FastAI_StarterScript.ipynb. Is that your intention?

**Comment by BrunoGrandePhD** [comment]
Given your suggestion in #216, I propose that we close this PR and add both notebooks to the new repo in the `notebooks/` subfolder with documentation pointing to `2_FastAI_StarterScript.ipynb` in this repo. Once that repo is up and running, we can update the documentation in `ModelTraining` with a link to the new training code.

**Comment by dthaler** [comment]
@pastorep @bnestor do you agree with closing this PR?
